pkg/buffer_config/buffer_config.go

```diff
@@ -33,10 +33,12 @@ const (
 	MinimalMultiplier           = 0.0
 	MinimalRandomizationFactor  = 0.0
 	MinimalRetryShutdownTimeout = 2 * MinimalMaxElapsedTime
+	PurgeToLifetimeRatio        = 3
```

Comment:
Would be good to add a comment on what this constant represents.

Reply:
Added